framework/src/common/errors.rs

```diff
@@ -71,3 +78,68 @@ pub enum NetBricksError {
     #[doc(hidden)]
     __Nonexhaustive,
 }
+
+#[macro_export]
+macro_rules! error_chain {
+    ($error:expr) => {
+        $crate::common::errors::print_chain($error, log::LevelFilter::Error, true)
+    };
+    () => {};
```

Comment:
Is this needed? Not sure how I feel about a macro that can resolve to a no op.

Reply:
I'll double check, probably can remove.